api/api_test.go

```diff
@@ -278,6 +296,12 @@ func TestAPI(t *testing.T) {
 		{"remove pending-ineligible", data1, failNone, api.Status, "GET", "?domain=removal-pending-ineligible.test",
 			200, jsonContentType, wantBody{state: &database.DomainState{
 				Name: "removal-pending-ineligible.test", Status: database.StatusPending}}},
+		{"remove automated pending-eligible", data1, failNone, api.Status, "GET", "?domain=automated-removal-pending-eligible.test",
+			200, jsonContentType, wantBody{state: &database.DomainState{
+				Name: "automated-removal-pending-eligible.test", Status: database.StatusPendingRemoval}}},
+		{"remove automated pending-ineligible", data1, failNone, api.Status, "GET", "?domain=automated-removal-pending-ineligible.test",
+			200, jsonContentType, wantBody{state: &database.DomainState{
+				Name: "automated-removal-pending-ineligible.test", Status: database.StatusPending}}},
```

Comment:
Based on the name automated-removal-pending, I'd expect at least one of these to have a status of StatusPendingAutomatedRemoval.